bpdm-gate/src/main/resources/db/migration/V4_0_0_5__make_legal_form_id_optional.sql

```diff
@@ -0,0 +1,3 @@
+-- Description: Make 'legal_form_id' column optional in the 'legal_entities' table
+-- Alter the 'legal_form_id' column to allow NULL values
+ALTER TABLE bpdmgate.legal_entities ALTER COLUMN legal_form_id DROP NOT NULL;
```

Comment:
Please remove the schema name here. Without the explicit schema name we would be more flexible in the configuration on where these migration files are applied.

Reply:
Removed schema name